free-2.md

```diff
@@ -58,11 +79,18 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+- I could see it being much more difficult to implement outside of language compared to idea 1. Its also because I have less experience with pixel art. However, since everything is pixel by pixel, it is much easier to loop through/ access specific pixels. So that it has over a general art generation program. 
+
 ### Scope
 
 _How big an idea is this? How ambitious is this project?_
 
+- Id say its more ambitous then idea 1. However it has a lot of "checkpoints" that could just be the entire project. 
+
 ### Benefits and drawbacks
 
 _Why might this be a good idea for a project? Why might this not be a good idea
 project?_
+
+- pros : its art. its using pixels. Lets me dive into pixel art. pixel manipulation can be easy 
+- cons : hard to justifiy compared to just general art programs. Image manipulation can be dififuclt. 
```

Comment:
This kind of reminds me of animation programs where you can rig characters for animation. I don't think there is anything that can rig a pixelated character and automatically generate sprite sheets.